js/sum.js

```diff
@@ -0,0 +1,46 @@
+let addBtn = document.getElementById('add-btn');
+let newNum = document.getElementById('new-num');
+let numsContainer = document.getElementById('added-nums-container');
+let sumElem = document.getElementById('sum');
+let isOdd = false;
+
+addBtn.addEventListener('click', () => {
+    if (newNum.value === '') {
+        alert('Enter a number, please!');
+        return;
+    }
+
+    isOdd = !isOdd;
+    let currentElem = document.createElement('div');   
+    isOdd ?  currentElem.setAttribute('class', 'current-added-elem odd') : currentElem.setAttribute('class', 'current-added-elem');  
+    let addedNum = document.createElement('span');
+    let deleteBtn = document.createElement('button');
+    deleteBtn.style.all = 'unset';
+    deleteBtn.innerText = '(delete)';
+    deleteBtn.setAttribute('id', 'delete');
+    deleteBtn.setAttribute('class', 'delete-btn'); 
+    let textNode = document.createTextNode((+newNum.value).toFixed(2));
```

Comment:
It's not a good place to change/cut the data. The data should be limited/validated before this point of business logic.